integration/test/basic/project.yaml

```diff
@@ -7,7 +7,7 @@ test:
     - "REGISTRY_PULL_SECRET=${REGISTRY_PULL_SECRET}"
     - "CLUSTER_NAME=${CLUSTER_NAME}"
     - "VAULT_TOKEN=mytoken"
-    - "COMMON_DOMAIN_GUEST=${COMMON_DOMAIN_GUEST}"
+    - "COMMON_DOMAIN=${COMMON_DOMAIN}"
```

Comment:
I've vendored e2e-harness and updated the env var to align with aws-operator. Hopefully this will fix e2e.

Reply:
Ok, I've already added `COMMON_DOMAIN` to circleci env, not sure if changing the name of the awsconfig resource chart to apiextensions is also required https://github.com/giantswarm/cluster-operator/commit/b34b0fa8a3c4ec6e651d1a1ea1f794c28d9ddffd. The build is running, let's see.

Reply:
Thanks Fede!

I'd missed using the chart from apiextensions. I've added here too. I'll run e2e once we know how your build went.

Reply:
It was still failing, turns out that aws-operator was complaining about a missing `credential-default` secret:
```
```

Reply:
Master is green, merging it here should fix tests.

Reply:
Sorry for the mess introduced. I missed e2e-harness upgrade here.
Thanks for sorting <3

Reply:
@fgimenez Yes thanks for sorting! <3 I'll rebase and re-run e2e.